packages/open-normal-extension/fi.ssrc.open_normal.json

```diff
@@ -0,0 +1,11 @@
+// SPDX-FileCopyrightText: 2022-2024 TII (SSRC) and the Ghaf contributors
```

Comment:
License header could be moved to own `fi.ssrc.open_normal.json.license` or even attributed via `.reuse/dep5` file